bootstrap_form.gemspec

```diff
@@ -9,20 +8,25 @@ Gem::Specification.new do |s|
   s.version     = BootstrapForm::VERSION
   s.authors     = ["Stephen Potenza", "Carlos Lopes"]
   s.email       = ["potenza@gmail.com", "carlos.el.lopes@gmail.com"]
-  s.homepage    = "http://github.com/bootstrap-ruby/rails-bootstrap-forms"
-  s.summary     = "Rails form builder that makes it easy to style forms using Twitter Bootstrap 3+"
-  s.description = "bootstrap_form is a rails form builder that makes it super easy to create beautiful-looking forms using Twitter Bootstrap 3+"
+  s.homepage    = "https://github.com/bootstrap-ruby/rails-bootstrap-forms"
+  s.summary     = "Rails form builder that makes it easy to style forms using "\
+                  "Bootstrap 4"
+  s.description = "bootstrap_form is a rails form builder that makes it super "\
+                  "easy to create beautiful-looking forms using Bootstrap 4"
+  s.license     = "MIT"
 
-  s.files = Dir["{app,config,db,lib}/**/*"] + ["MIT-LICENSE", "Rakefile", "README.md"]
-  s.test_files = Dir["test/**/*"]
+  s.files = `git ls-files -z`.split("\x0").reject do |f|
```

Comment:
I like the idea of automating the generation of the file list. This looks to me like it will include in the gem some of the files that I would think of as development files, like `.gitignore` and `.travis.yml`. I don't think it's a problem, but I'm wondering  there's a reason to include these files.

Reply:
It does include some unnecessary stuff, but I like that it is automatic so that we are less likely to forgot to add a file in the future. It is a standard suggested by Bundler, so I feel it is safe to follow:

https://github.com/bundler/bundler/blob/fa7e4640c06d6c7d9e7c99480342f34008aa05fd/lib/bundler/templates/newgem/newgem.gemspec.tt#L31-L33